utils/data_processing/recode_csv.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
I would like to have @saif-analyze or @mariobecerra 's opinion here since I don't know how useful this feature is for you guys (I do not have the knowledge to judge that).

Reply:
@saif-analyze